data/pam.d/pacemaker-base

```diff
@@ -0,0 +1,3 @@
+%PAM-1.0
+auth       include  password-auth
+account    include  password-auth
```

Comment:
Have you seen daemons/based/cib.pam? It appears this idea was already considered at one time. I'm guessing we should remove that with this change.

The current cib.pam contents appear to be Debian-oriented, which brings up the question that different distros name the default modules differently -- this file may need to vary by distro.

@gao-yan , is SUSE using cib.pam or anything similar downstream? It looks like cib.pam was added for SUSE. Do you have any thoughts on the default modules here?

Reply:
Ah, `grep` didn't discover this or I missed that.

Reply:
Btw. had a look at OpenSUSE packages and didn't find that there.

Reply:
It's not shipped by SUSE indeed. The default pam service was switched from "cib" to "login" as of: https://github.com/ClusterLabs/pacemaker/commit/b17fc33d0a6c5e97de431b5a80c3b7e10c7c8957

I don't know the story behind this though ... Given that there's the option "CIB_pam_service" which could be used for customization, may I ask why a separate pam service file is needed here?

Reply:
Thinking about this some more, I do think we need to keep "login" as the default, to maintain backward compatibility in case any sites have modified "login" and are depending on it to work for pacemaker. (Though I don't think remote CIB access is widely used, and I'd love to drop support for it eventually -- GUIs are mature enough now to better cover that use case, I think.)

We can still supply a preferred PAM service file and document it with CIB_pam_service in sysconfig.

I also would like to go with "pacemaker" as the name, since I don't foresee any use case where different PAM services are desired for different pacemaker components, and any users who desire something that complicated can figure it out on their own.